corehq/form_processor/parsers/ledgers/form.py

```diff
@@ -145,11 +145,8 @@ def _extract_ledger_nodes_from_xml(node):
 
     for elem in _extract_ledger_nodes_from_xml(form_xml):
         report_type, ledger_json = convert_xml_to_json(elem, last_xmlns=COMMTRACK_REPORT_XMLNS)
-
-        # apply the same datetime & string conversions
-        # that would be applied to XFormInstance.form
-        adjust_datetimes(ledger_json)
-        ledger_json = XFormInstance({'form': ledger_json}).form
+        if ledger_json.get('@date'):
+            ledger_json['@date'] = adjust_text_to_datetime(ledger_json['@date'])
```

Comment:
what happens if this raises a `iso8601.ParseError`?

Reply:
It would result in a 500.  I suppose you could have get there if a user submitted a date like April 31.